setup.py

```diff
@@ -31,13 +25,13 @@
     'license': "BSD",
     'author': "HERA Team",
     'author_email': "hera-sw@lists.berkeley.edu",
-    'version': version.version,
+    "use_scm_version": {"local_scheme": branch_scheme},
     'packages': ['hera_mc', 'hera_mc.tests'],
     'scripts': glob.glob('scripts/*'),
     'include_package_data': True,
     'install_requires': ["numpy", "astropy", "sqlalchemy", "psycopg2",
                          "alembic", "python-dateutil", "tabulate", "pandas",
-                         "psutil", "cartopy", "pyyaml", "redis"],
+                         "psutil", "cartopy", "pyyaml", "redis", "setuptools_scm"],
```

Comment:
Do we actually need `setuptools_scm` as an install dependency? I know it is a build dependency and we have that declared in the `pyproject.toml`, but I wonder if it is necessary here or not.

Reply:
I think we need it as an install dependency because it's imported in the `__init__`.py.

Reply:
fair enough! :D let's get this done.